### DISCUSSION TITLE
Can osu! Lazer add hitbox control for mobile? Just like FNF. I can't play with wide skin

### DISCUSSION THREAD

**Discussion by Fikrienz** [ppy/osu#26741] [Ideas] [answered]
![Screenshot_2024-01-27-14-15-53-12_131c8b06131edeb2ecc729e7bbaafce9](https://github.com/ppy/osu/assets/157803695/3e05b992-0288-4c8b-abfc-e80747e6f5d4)

https://github.com/ppy/osu/assets/157803695/3a0d0ca6-bac4-4275-b37f-60f7afc9b0ea

**Comment by Flamiii** [comment, marked as answer]
I'm fairly certain this is planned (https://github.com/ppy/osu/issues/18882). There will probably be a screen area larger than the columns of your skin that can be used for input so you don't have to play with a skin with extremely wide columns.
